electron_app/electron.ts

```diff
@@ -782,6 +782,15 @@ function getHelpMenu(): MenuItem {
         electron.shell.openExternal(documentationUrl);
       },
     },
+    {
+      label: 'BPMN Element Documentation',
```

Comment:
Would prefer a label like:

```suggestion
      label: 'Supported BPMN Elements',
```

Reply:
Find die erste Variante besser. 😉